src/database/seeders/20240428145740-demo-user.js

```diff
@@ -1,3 +1,42 @@
+// const bcrypt = require("bcrypt");
+// const hashPassword = async (password) => {
+// 	const salt = await bcrypt.genSalt(10);
+// 	const hashedPass = await bcrypt.hash(password, salt);
+// 	return hashedPass;
+// };
+// ("use strict");
+// /** @type {import('sequelize-cli').Migration} */
```

Comment:
Since this lines are commented, i don't think you need them. You can remove them. Apart from that everything is great